src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/ParametersValidatorUtil.java

```diff
@@ -36,6 +54,13 @@ static void checkIsNotNullOrEmptyString(String stringToValidate,
         }
     }
 
+    static void checkLength(String stringToValidate, String jobParametersName) throws JobParametersInvalidException {
```

Comment:
It would be more consistent to check also for length 0 (empty string) in this method, and leave the null only for that specific value.